### PR TITLE
Update .travis config

### PR DESCRIPTION
**Pull request by othercorey**
Don't set DEFAULT when running CHECKS config.

### CODE REVIEW

**Comment by ADmad** [comment]
It seems the the test suite is still run twice for the coverage job.

File: composer.json [hunk 1]
```diff
@@ -43,12 +43,6 @@
           "phpunit",
           "@reset-ruleset"
         ],
-        "test-coverage": [
```
Comment (ADmad) [inline composer.json]: This can be retained to conveniently run test coverage locally if needed.

Comment (othercorey) [reply composer.json]: It doesn't work without other packages and another script. I'd rather just remove what doesn't work.

Comment (ADmad) [reply composer.json]: ok

**Pull request merged**